### PR TITLE
add packages: ko, apko, melange

### PR DESCRIPTION
**Pull request by imjasonh**
Signed-off-by: Jason Hall <jason@chainguard.dev>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -185,5 +185,8 @@ $(eval $(call build-package,fontconfig,2.14.0-r0))
 $(eval $(call build-package,openjdk,17.0.4.1-r0))
 $(eval $(call build-package,su-exec,0.2-r0))
 $(eval $(call build-package,postgresql,11.17-r0))
+$(eval $(call build-package,ko,0.12.0-r0))
+$(eval $(call build-package,apko,0.5.0-r0))
+$(eval $(call build-package,melange,0.1.0-r0))
```
Comment (kaniini) [inline Makefile:190]: Is it worth packaging this version of melange, considering it is too old to build Wolfi?

Comment (imjasonh) [reply Makefile:190]: I can drop it if you want. I thought it'd be nice to start having it packaged, and we can bump it when we do the next melange release. If you'd prefer to wait until 0.2.0 I can do that too.

Comment (kaniini) [reply Makefile:190]: We can just hold until apko 0.6 and melange 0.2 are shipped, which i might do tomorrow. :)

**Comment by tuananh** [comment]
@imjasonh should we copy the melange pipelines to /usr/share/melange as well?

**Comment by imjasonh** [comment]
> @imjasonh should we copy the melange pipelines to /usr/share/melange as well?

Starting soon™️ , melange will compile in the built-in pipelines, so this won't need to install into `/usr/share/melange/pipelines` 🚀

**Comment by tuananh** [comment]
> > @imjasonh should we copy the melange pipelines to /usr/share/melange as well?
> 
> Starting soon™️ , melange will compile in the built-in pipelines, so this won't need to install into `/usr/share/melange/pipelines` 🚀 

Niceee.

If i want to add new pipeline in melange, what's the recommended way now?

I still want to use the default pipelines, just add some new one

**Comment by imjasonh** [comment]
> If i want to add new pipeline in melange, what's the recommended way now?
> 
> I still want to use the default pipelines, just add some new one

If you want to add a new built-in pipeline, for all users of Melange, propose them here: https://github.com/chainguard-dev/melange/tree/main/pkg/build/pipelines

If you want to define pipelines only for your own use, put them anywhere in your local filesystem, and invoke `melange build --pipeline-dir=/path/to/my/pipelines`.

**Comment by imjasonh** [comment]
Closing for now.

**Pull request closed without merging**